person.rb

```diff
@@ -1,15 +1,16 @@
-require './nameable'
+require_relative 'nameable'
 
 class Person < Nameable
-  attr_accessor :id, :name
+  attr_reader :rentals, :name
 
   def initialize(id, name)
     super()
     @id = id
     @name = name
+    @rentals = [] # To store rentals
   end
 
-  def correct_name
-    @name
+  def add_rental(rental)
+    @rentals << rental
   end
```

Comment:
For the same reason as stated above, kindly update your add_rental method. take a look at the suggestion below😉.

```suggestion
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```